src/Native/System.Net.Security.Native/pal_gssapi.h

```diff
@@ -0,0 +1,141 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#pragma once
+
+#if HAVE_GSSFW_HEADERS
+typedef struct gss_name_t_desc_struct GssName;
```

Comment:
@kapilash I'm confused.  The code says that `gss_name_t` is a typedef for the struct type, but your comment here says it's for the struct-pointer type.

I'm assuming that it's just the struct type, otherwise I'd have to ask how you're supposed to properly create one of these things.